### ISSUE TITLE
Features for Version 1

### ISSUE THREAD

**Issue by BarkleyBG** [BarkleyBG/LCC#22]
This is migrating the open issues from #18. We can close the issue on auto-building the PDF for now, as that is not of imminent interest.

Any PDF should have

- [ ] title page #5 
- [ ] front matter including
    - [ ] contributor information #5 
    - [ ] date and version #21
    - [ ] information on how to contribute (i.e. link to the repo) #19 and #14
    - [ ] introduction to the document and its aims 
- [x] ToC #6  (references and index can be added later so i'm not even adding a bullet point)

I'm thinking we should follow the git repo's model of master/next/pu branches. For us, maybe we could user master/next_update/distant_future?

- Any changes to next_update would need to get merged into distant_future. That should be easy enough.
- topic branches would be used to make changes to next_update and distant_future. Hotfix topic branches of course for master too.
- The one thing I don't know is how we would add a watermark to the PDF built from next_update that gets shared with readers for proofreading. 
    - Maybe we could have a next_update_mirror branch with the watermark on it? 
    - I worry that this would result in both the next_update and next_update_mirror branches would be changing in parallel. Perhaps we could simply merge often to keep things simple? 
    - Alternatively we could build the PDF in a commit in the next_update branch, mark it with a tag, and then delete the watermark in the next commit and continue forward? Hmmm...
